### PR TITLE
Added avc,prk and 4k profiles

### PR DESCRIPTION
**Pull request by erickyun**

### CODE REVIEW

**Comment by DavidBuchanan314** [comment]
Hi, thanks for the contribution!

I don't have time to test it myself right now, but if you can tell me which browser(s) and platforms you've tested it on, I'm willing to take your word for it and merge

File: src/cadmium-playercore-shim.js [hunk 1]
```diff
@@ -53,6 +51,52 @@ function get_profile_list() {
 		"BIF320"
 	];
 
+        if (my_config["use_avc.high"]) {
```
Comment (DavidBuchanan314) [inline src/cadmium-playercore-shim.js:54]: It looks like you're mixing tabs and spaces here, please use tabs for indentation throughout

**Comment by erickyun** [comment]
> Hi, thanks for the contribution!
> 
> I don't have time to test it myself right now, but if you can tell me which browser(s) and platforms you've tested it on, I'm willing to take your word for it and merge

Test on Windows 11 Edge browser sir.
I appreciate your great work, and thank you for this repo.

File: src/cadmium-playercore-shim.js [hunk 2]
```diff
@@ -61,7 +105,31 @@ function get_profile_list() {
...
 	}
 
-	if (my_config["use_5.1"]) {
+	if (my_config["use_ddplus-5.1"]) {
+		custom_profiles = custom_profiles.concat([
+			"ddplus-5.1-dash",
+		]);
+	}
+
+	if (my_config["use_ddplus-5.1"]) {
```
Comment (DavidBuchanan314) [inline src/cadmium-playercore-shim.js:114]: `use_ddplus-5.1` is done twice

**Comment by DavidBuchanan314** [comment]
ty!

**Comment by kadrim** [comment]
sorry to step in, but this addition broke playback in firefox. in edge it working fine, but FF refuses to playback videos at all

**Comment by DavidBuchanan314** [comment]
Oop, I thought that might happen. Try fiddling with the config options

**Comment by DavidBuchanan314** [comment]
Rolled back for now

**Comment by erickyun** [comment]
> sorry to step in, but this addition broke playback in firefox. in edge it's working fine, but FF refuses to playback videos at all
> 
> error code f7355-1203
> 
> OS: Windows 11

As @DavidBuchanan314 said try changing [config options](https://github.com/erickyun/Netflix-Playback-Features/blob/6812298d42f458ec1751dd43baedfc73e60cf2f3/src/cadmium-playercore-shim.js#L4) to from true to false and vice versa also firefox&chrome do only support 720p [acoording to NF link](https://help.netflix.com/en/node/30081) right?
 So avc,prk wouldn't work if it is set to true.

**Pull request merged**